test/test-mongocrypt-kms-responses.c

```diff
@@ -65,6 +65,31 @@ _test_one_kms_response (_mongocrypt_tester_t *tester, bson_t *test)
          ctx = mongocrypt_ctx_new (crypt);
          mongocrypt_binary_destroy (bin);
          bin = _mongocrypt_tester_encrypted_doc (tester);
+         tester->key_file_path = "./test/example/key-document.json";
+         ASSERT_OK (mongocrypt_ctx_decrypt_init (ctx, bin), ctx);
+      } else if (0 == strcmp ("azure_oauth_datakey", ctx_type)) {
+         ctx = mongocrypt_ctx_new (crypt);
```

Comment:
Hoist this up above the if/else since it just copy/paste

Reply:
Done.